sdk.adoc

```diff
@@ -0,0 +1,111 @@
+= uProtocol SDK Requirements
+:toc:
+:sectnums:
+
+The key words "*MUST*", "*MUST NOT*", "*REQUIRED*", "*SHALL*", "*SHALL NOT*", "*SHOULD*", "*SHOULD NOT*", "*RECOMMENDED*", "*MAY*", and "*OPTIONAL*" in this document are to be interpreted as described in https://www.rfc-editor.org/info/bcp14[IETF BCP14 (RFC2119 & RFC8174)]
+
+----
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+----
+
+== Overview
+The purpose of uProtocol language specific SDKs are to provide the consistent mental model across all programming languages (Java, C++, Rust, Python, etc...) for core uP-L1 & L2 specifications, interfaces, and data models. The SDK is core to all uProtocol software as it is used to implements uEs (apps & services), transports, and platforms 
```

Comment:
SDK is not only libraries.
You can state these are SDK libraries.

Reply:
Rephrased and removed the term SDK like we discussed